data/members/Finlay_Clark.yaml

```
@@ -0,0 +1,10 @@
+name: Finlay Clark
+role: Researcher
+filter-role: External Collaborators
```

```suggestion
lab: Cole Group
filter-role: External Collaborators
```
And confirming he's not part of the "Scientific Staff" role I see others grouped within; I don't know what the difference is

---

Yeah, it's not clear to me what qualifies someone as "Scientific Staff." Probably means our filter-roles need an overhaul.